src/css/main.less

```diff
@@ -1478,3 +1472,7 @@ h3.small-caps {
     padding: 0;
   }
 }
+
+.task-details dt {
+  text-align: left;
```

Comment:
@aldipower I tried using Bootstrap's `text-left` class but nope. If you have a better idea shoot it!